src/Sluggable/SluggableListener.php

```diff
@@ -257,7 +257,7 @@ public function prePersist(EventArgs $args)
         if ($config = $this->getConfiguration($om, $meta->getName())) {
             foreach ($config['slugs'] as $slugField => $options) {
                 if ($meta->isIdentifier($slugField)) {
-                    $meta->getReflectionProperty($slugField)->setValue($object, '__id__');
+                    $meta->getReflectionProperty($slugField)->setValue($object, '__sluggable_placeholder_'.random_int(1, PHP_INT_MAX).'__');
```

Comment:
How about using `uniqid()` instead of `random_int()`? I believe the former is created for exact this purpose - there needs to be uniqueness, but not necessarily cryptographic security. Chances of collision with `random_int` are practically 0, but still a change to consider.

The call would probably also get a bit cleaner: `uniqid('__sluggable_placeholder_')` (`'__'` at the end is unused, so not necessary)

Reply:
Yeah, that'd be fine.  Admittedly I went with the first solution that came to mind, which was `random_int()`, but that is a bit overkill here since it doesn't need cryptographic-backed uniqueness.